### ISSUE TITLE
Не работают подсказки для "Configuration arrays for object instantiation"

### ISSUE THREAD

**Issue by beeryukov** [nvlad/yii2support#275]
### What steps will reproduce the problem?
Объявить класс, наследующий yii\base\BaseObject (в котором реализована фича configuration arrays);
Вызвать конструктор класса через new;
Написать пустые квадратные скобки в вызове конструктора (таким образом начать заполнять configuration array);

### What is the expected result?
В качестве ключей массива появляются подсказки с именами свойств класса

### What do you get instead?
Подсказки не появляются. Всплывает сообщение "No suggestions"

### Additional info
Сам плагин активен и работает. В частности работают подсказки для Model::rules и для Url::to()

| Q                | A
| ---------------- | --------
| IDE Name         | PhpStorm 
| IDE Version      | 2021.1.2 (Build #PS-211.7142.44, built on April 30, 2021)
| Plugin version   | com.yii2support (0.10.58.37)
| Yii App Template | basic

**Comment by beeryukov** [comment]
Указанные подсказки работают в конфиге, а также в контроллере в функции behaviors, например:

```
'authManager' => [
       'class' => 'yii\rbac\DbManager',
        'defaultRoles' => ['guest', 'user'],
 ]
```

Не работает только в сочетании с
```
new SomeClass([
]);
```

**Comment by iglooom** [comment]
@beeryukov теста ради попробуй вот эту версию, я там правил кучку всяких мелочей [Yii2 Support-0.10.58.40.zip](https://github.com/nvlad/yii2support/files/6694020/Yii2.Support-0.10.58.40.zip)

**Comment by beeryukov** [comment]
@iglooom, спасибо, попробовал. Подсказки для `new SomeClass([])` появились, уже хорошо :)
А вот `Go To Declaration, Rename, Find usages` не заработало.

**Comment by gvlasov** [comment]
https://plugins.jetbrains.com/plugin/9388-yii2-support

> Calculates return type for Yii::createObject call and one/all method calls of ActiveRecord

По какой-то причине автоподсказки для создаваемых через new объектов начинаются не с `BaseObject`, а с `ActiveRecord`. Причины не понимаю, хотелось бы и для `BaseObject` это иметь, но раз не сделали, то возможно был веский повод. Хотя тогда странно, что с `Yii::createObject` работает. @nvlad, запили пожалуйста!

**Comment by iglooom** [comment]
@gvlasov а речь про тот билд что я скинул?

**Comment by gvlasov** [comment]
@iglooom Нет, хочется чтобы автор плагина добавил фичи из того билда в основную ветку

**Comment by iglooom** [comment]
Да, тоже хотелось бы, я даже на указанное мыло писал уже ему, но пока без ответа.

**Comment by nssmart** [comment]
> @beeryukov теста ради попробуй вот эту версию, я там правил кучку всяких мелочей [Yii2 Support-0.10.58.40.zip](https://github.com/nvlad/yii2support/files/6694020/Yii2.Support-0.10.58.40.zip) Это билд из этого PR #270

Большое спасибо, как я устал модели делать вот так
$model = new Model(); $model->abc = '';

С этим билдом вновь работает
new Model(['...

Спасиииибо!)
